pkg/lightning/config/config.go

```diff
@@ -152,7 +158,6 @@ func (cfg *Config) ToTLS() (*common.TLS, error) {
 
 type Lightning struct {
 	TableConcurrency  int    `toml:"table-concurrency" json:"table-concurrency"`
-	IndexConcurrency  int    `toml:"index-concurrency" json:"index-concurrency"`
```

Comment:
removing this breaks existing `tidb-lightning.toml` which defined `index-concurrency` (triggers "config file contained unknown configuration options" error).

Reply:
For the current logic, remove index-concurrency may bring a side-effect. Since we start import index engine only after all the data engine are imported. I'm afraid after this change, more table's index engine will finally start import at the same time than before, which may hurt the performance.

Reply:
But most of them will be blocked by TableConcurrency

Reply:
I have revert it.